devices/ePaper/Primitives/Color.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) 2022 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+
+namespace Iot.Device.ePaper.Primitives
+{
+    /// <summary>
+    /// A color data structure for RGB colors.
+    /// </summary>
+    public struct Color
```

Comment:
We do have a System.Drwaing with color as a nuget, you should use it, it has quite some predefined colors: https://github.com/nanoframework/nanoFramework.IoT.Device/blob/develop/devices/System.Drawing/System.Drawing.cs